### ISSUE TITLE
[Bug] Baidu Map cannot work properly

### ISSUE THREAD

**Issue by TPLeung** [darkreader/darkreader#7862]
# Bug Report

## Bug Description

The dark reader can convert the map into dark mode, but the search bar is not working and the function buttons are disappeared on the upper-right corner. An error occurs in the console window.
![image](https://user-images.githubusercontent.com/18712884/148180690-dc358b91-227f-4c7a-8838-837b8ada7461.png)

## Website Address

https://maps.baidu.com/

## Steps To Reproduce

- Go to https://maps.baidu.com/ with dark reader default on.
- The search bar cannot work properly, also some functional buttons on the upper-right corner disappear
- Turn off the dark reader for this site only, and reload the website, everything works fine in light mode.

## Expected Behavior

See description before.

## Actual Behavior

Returning the following error:
```
mod_5dbb339.js:1 Uncaught TypeError: Cannot read properties of undefined (reading 'getAttribute')
    at n (mod_5dbb339.js:1)
    at Function.require.loadCss (mod_5dbb339.js:1)
    at Object.<anonymous> (Vcode_d33db8a.js:1)
    at require (mod_5dbb339.js:1)
    at Object.<anonymous> (init-pkg_7c03cb4.js:18)
    at require (mod_5dbb339.js:1)
    at Object.<anonymous> (init-pkg_7c03cb4.js:5)
    at require (mod_5dbb339.js:1)
    at Object.<anonymous> (init-pkg_7c03cb4.js:15)
    at require (mod_5dbb339.js:1)
```

## Screenshots

![image](https://user-images.githubusercontent.com/18712884/148181149-523142d1-4eb0-47f6-9f0c-60129dd0fef5.png)
![image](https://user-images.githubusercontent.com/18712884/148181015-dcc9507a-2cc7-4754-8543-c45afd7765ed.png)
After turning into dark mode:
1. The search bar can type things, but cannot search things.
2. The function button disappeared, something for showing the road condition, account log in, etc..

## System Information

- OS: Windows 10/11 
- Browser: Chrome 96
- Dark Reader Version: 4.9.43

**Comment by Gusted** [comment]
Already fixed in the next release.

**Comment by TPLeung** [comment]
> Already fixed in the next release.

Still exist in version 4.9.44, not sure if it's fixed in version 5.0.
